client/src/main/java/org/jboss/fuse/mvnd/client/Installer.java

```diff
@@ -159,11 +164,15 @@ static void unzip(Path localZip, Path destinationDir) {
     static Path download(URI zipUri) {
         try {
             final Path localZip = Files.createTempFile("", "-mvnd-dist.zip");
-            LOG.debug("Downloading {} to {}", zipUri, localZip);
+            if (LOG.isInfoEnabled()) {
+                LOG.info("Downloading {}", zipUri);
+            } else {
+                LOG.debug("Downloading {} to {}", zipUri, localZip);
+            }
             try (
                     InputStream in = new BufferedInputStream(zipUri.toURL().openStream(), BUFFER_SIZE);
                     OutputStream out = new BufferedOutputStream(Files.newOutputStream(localZip), BUFFER_SIZE)) {
-                copy(in, out);
+                copy(in, out, LOG.isInfoEnabled());
```

Comment:
Shouldn't that be `isDebugEnabled()` so that the verbosity is greater in `debug` mode than in `info` mode ?

Reply:
The idea is on this particular location is to suppress the download progress output by setting the log level to WARN or ERROR. So I think the code is as intended here.